client/state/sites/selectors.js

```diff
@@ -71,6 +71,22 @@ export const getSite = createSelector(
 			domain: getSiteDomain( state, siteId ),
 			is_previewable: isSitePreviewable( state, siteId )
 		};
+
+		if ( attributes.jetpack ) {
```

Comment:
Can we move this into a helper function:

```
getJetpackComputedAttributes( site ) {
    if ( ! site.jetpack ) {
        return {};
    }
    return {
        canManage: canJetpackSiteManage( state, siteId ),
       //...
    }
);
```

And then in this method we can do:
```
return {
			...site,
			...getComputedAttributes( site ),
        ...getJetpackComputedAttributes( site ),
			hasConflict: isSiteConflicting( state, siteId ),
			title: getSiteTitle( state, siteId ),
			slug: getSiteSlug( state, siteId ),
			domain: getSiteDomain( state, siteId ),
			is_previewable: isSitePreviewable( state, siteId )
		};
```